contracts/src/periphery/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierBase, IEigenDACertVerifier} from "src/periphery/cert/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDACertVerifierRouter} from "src/periphery/cert/interfaces/IEigenDACertVerifierRouter.sol";
+import {OwnableUpgradeable} from "lib/openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+
+contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgradeable {
+    /// @notice A mapping from an activation block number (ABN) to a cert verifier address.
+    mapping(uint32 => address) public certVerifiers;
+
+    /// @notice The list of Activation Block Numbers (ABNs) for the cert verifiers.
+    /// @dev The list is guaranteed to be in ascending order
+    ///      and corresponds to the keys of the certVerifiers mapping.
+    uint32[] public certVerifierABNs;
+
+    event CertVerifierAdded(uint32 indexed activationBlockNumber, address indexed certVerifier);
+
+    error ABNNotInFuture(uint32 activationBlockNumber);
+    error ABNNotGreaterThanLast(uint32 activationBlockNumber);
+    error InvalidCertLength();
+
+    /// IEigenDACertVerifierRouter ///
+
+    /// @inheritdoc IEigenDACertVerifierBase
+    function checkDACert(bytes calldata abiEncodedCert) external view returns (uint8) {
+        return IEigenDACertVerifier(getCertVerifierAt(_getRBN(abiEncodedCert))).checkDACert(abiEncodedCert);
+    }
+
+    function getCertVerifierAt(uint32 referenceBlockNumber) public view returns (address) {
+        return certVerifiers[_findPrecedingRegisteredABN(referenceBlockNumber)];
+    }
+
+    /// ADMIN ///
+
+    function initialize(address _initialOwner, address certVerifier) external initializer {
+        _transferOwnership(_initialOwner);
+        // Add a default cert verifier at block 0, which will be used for all blocks before the first ABN.
+        _addCertVerifier(0, certVerifier);
```

Comment:
Perhaps adding an initial cert verifier should be optional? Right now I guess you could just pass in a zero address and it would be fine, but this would need additional consideration if we decide to check that the address passed to `_addCertVerifier` implements the expected interface